packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -1324,6 +1323,234 @@ export class CodeModelBuilder {
     }
   }
 
+  private processParameterGrouping(
+    op: CodeModelOperation,
+    sdkMethod: SdkServiceMethod<SdkHttpOperation>,
+    bodyParameterFlattened: boolean,
+  ) {
+    const httpOperation = sdkMethod.operation;
+    const methodSignatureOverridden = sdkMethod.decorators.some(
+      (it) => it.name === "Azure.ClientGenerator.Core.@override",
+    );
+
+    if (methodSignatureOverridden) {
+      // limit the effect of "SdkServiceMethod"
+      // only process it and its parameters, when the "@override" is defined on the operation
+      this.processSdkMethodOverride(op, sdkMethod);
```

Comment:
New code here.

To limit the impact, I only handle the `sdkMethod.parameter` for grouping, when there is `@override` decorator on the method.